dspace-api/src/main/java/org/dspace/content/virtual/Concatenate.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.virtual;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * A bean implementing the {@link VirtualBean} interface to achieve the generation of Virtual metadata
```

Comment:
This Javadoc doesn't seem to describe "Concatenate".

Reply:
The JavaDocs have been included